src/R/Actions/Impl/Utility/RInstallation.cs

```diff
@@ -259,7 +259,7 @@ public static class RInstallation {
         }
 
         public static Version GetRVersionFromFolderName(string folderName) {
-            if (folderName.StartsWith("R-")) {
+            if (folderName.StartsWith("R-", StringComparison.OrdinalIgnoreCase)) {
```

Comment:
:)

---
In reply to: [60325363](https://github.com/Microsoft/RTVS/pull/1648#discussion_r60325363) [](ancestors = 60325363)